common/OpTestSystem.py

```diff
@@ -328,10 +328,12 @@ def goto_state(self, state):
           raise unittest.SkipTest("OpTestSystem running QEMU/Mambo so skipping OpSystemState.OS test")
         if (self.state == OpSystemState.UNKNOWN):
           log.debug("OpTestSystem CHECKING CURRENT STATE and TRANSITIONING for TARGET STATE: %s" % (state))
+          log.debug(" *** UNKNOWN=0 OFF=1 IPLing=2 PETITBOOT=3 PETITBOOT_SHELL=4 BOOTING=5 OS=6 POWERING_OFF=7 UNKNOWN_BAD=8")
```

Comment:
Could we instead have a little helper that turns state numbers into strings and do
`log.debug("OpTestSystem CHECKING CURRENT STATE and TRANSITIONING for TARGET STATE: {} ({})".format(state, state_name(state))`?

Reply:
> a little helper

I thought about that as well, but in some places it just says "START STATE=xx" and for the consumer that is not familiar doesn't know the entire table so it is not quite as meaningfull, so I thought the entire state machine would help level set the terminology.